source/bson-binary-vector/bson-binary-vector.md

```diff
@@ -242,3 +244,7 @@ See the [README](tests/README.md) for tests.
         you want to store or transmit binary data more efficiently by grouping 8 bits into a single byte (uint8). For an
         example in Python, see
         [numpy.unpackbits](https://numpy.org/doc/2.0/reference/generated/numpy.unpackbits.html#numpy.unpackbits).
+
+## Changelog
+
+- 2025-02-04: Update validation for decoding into a FLOAT32 vector.
```

Comment:
Please add the date of the originally accepted spec:
- 2024-11-01: BSON Binary Subtype 9 accepted [DRIVERS-2926](https://jira.mongodb.org/browse/DRIVERS-2926) [(#1708)](https://github.com/mongodb/specifications/pull/1708[)

Reply:
BTW, do we have the convention of appending the ticket numbers? IMO, they are trackable from the git change log.

Reply:
The git change log contains all commits, not just those related to bson binary vectors. If we add just the PR, our convention is to include the jira ticket though.